### DISCUSSION TITLE
NAO CONSIGO REGISTRAR MINHA LICENÇA

### DISCUSSION THREAD

**Discussion by flavio085** [aclap-dev/video-downloadhelper#832] [Help and Q&A]
### Your question

Eu removi a extensão e depois instalei de novo, agora não consigo registrar a minha licença. 
Por favor, me ajudem!
Eu uso o firefox.

<img width="331" alt="A-MINHA-LICENÇA-NAO-ENTRA" src="https://github.com/aclap-dev/video-downloadhelper/assets/171888207/4475fcbe-1dcd-4c9c-8e26-552f42875468">

### Addon & CoApp version and details

Eu removi a extensão e depois instalei de novo, agora não consigo registrar a minha licença. 
Por favor, me ajudem!
Eu uso o firefox.

<img width="331" alt="A-MINHA-LICENÇA-NAO-ENTRA" src="https://github.com/aclap-dev/video-downloadhelper/assets/171888207/f998b7a2-5aae-451e-99e7-b28b044f8de8">

**Comment by jcfrog** [comment]
Hi,
Did you click on the "Check license key" link?
